### PR TITLE
Logging improvements

### PR DESCRIPTION
**Pull request by bhollis**
Took another pass at improving logging, especially around profile loads and DIM Sync operations. Some improved logging, some removed logging. The `TAG` convention is mostly just there to take some duplication out of the bundle.

### CODE REVIEW

File: src/app/i18n.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export function initi18n(): Promise<unknown> {
       {
         initImmediate: true,
         compatibilityJSON: 'v3',
-        debug: $DIM_FLAVOR === 'dev',
+        debug: false,
```
Comment (ryan-rushton) [inline src/app/i18n.ts:92]: Do we want this change? Or was it a change while debugging?

Comment (bhollis) [reply src/app/i18n.ts:92]: Yes, I don't think it's necessary.

**Review by nev-r — APPROVED** [review]
this seems working ok. changes are reasonable then the diff on d2-stores is a little wild.

File: src/app/inventory/d2-stores.ts [hunk 2]
```diff
@@ -158,26 +158,32 @@ function loadProfile(
...
         // Check to make sure the profile hadn't been loaded in the meantime
         if (getState().inventory.profileResponse) {
           cachedProfileResponse = getState().inventory.profileResponse;
         } else if (cachedProfileResponse) {
-          infoLog('d2-stores', 'Loaded cached profile from IndexedDB');
+          const profileAgeSecs =
+            (Date.now() - new Date(cachedProfileResponse.responseMintedTimestamp ?? 0).getTime()) /
+            1000;
+          infoLog(
+            'd2-stores',
```
Comment (delphiactual) [inline src/app/inventory/d2-stores.ts:176]: TAG

File: src/app/inventory/d2-stores.ts [hunk 3]
```diff
@@ -188,39 +194,63 @@ function loadProfile(
       // TODO: need to make sure we still load at the right frequency / for manual cache busts?
       cachedProfileMintedDate = new Date(cachedProfileResponse.responseMintedTimestamp ?? 0);
       const profileAge = Date.now() - cachedProfileMintedDate.getTime();
-      infoLog('d2-stores', `Cached profile is ${profileAge / 1000}s old.`);
       if (!storesLoadedSelector(getState()) && profileAge > 0 && profileAge < BUNGIE_CACHE_TTL) {
-        warnLog('d2-stores', 'Cached profile is new enough, skipping remote load.', profileAge);
+        warnLog(
+          'd2-stores',
```
Comment (delphiactual) [inline src/app/inventory/d2-stores.ts:199]: TAG

File: src/app/inventory/d2-stores.ts [hunk 4]
```diff
@@ -188,39 +194,63 @@ function loadProfile(
...
       if (cachedProfileResponse) {
+        const cachedProfileAgeSec = (now - cachedProfileMintedDate.getTime()) / 1000;
         if (remoteProfileMintedDate.getTime() <= cachedProfileMintedDate.getTime()) {
-          warnLog('d2-stores', 'Profile from Bungie.net is older than cached profile, discarding.');
+          const eq = remoteProfileMintedDate.getTime() === cachedProfileMintedDate.getTime();
+          const storesLoaded = storesLoadedSelector(getState());
+          const action = storesLoaded ? 'Skipping update.' : 'Using the cached profile.';
+          if (eq) {
+            infoLog(
+              'd2-stores',
```
Comment (delphiactual) [inline src/app/inventory/d2-stores.ts:222]: TAG

File: src/app/inventory/d2-stores.ts [hunk 5]
```diff
@@ -188,39 +194,63 @@ function loadProfile(
...
           // undefined means skip processing, in case we already have computed stores
           return storesLoadedSelector(getState())
             ? undefined
             : { profile: cachedProfileResponse, live: false };
         } else {
-          minimumCacheAge = Math.min(
-            minimumCacheAge,
-            remoteProfileMintedDate.getTime() - cachedProfileMintedDate.getTime(),
+          infoLog(
+            'd2-stores',
```
Comment (delphiactual) [inline src/app/inventory/d2-stores.ts:241]: TAG

File: src/app/inventory/d2-stores.ts [hunk 6]
```diff
@@ -188,39 +194,63 @@ function loadProfile(
...
+            `Profile from Bungie.net is is ${remoteProfileAgeSec}s old, while the cached profile is ${cachedProfileAgeSec}s old.`,
+            `Using the new profile from Bungie.net.`,
           );
-          infoLog('d2-stores', `Profile from Bungie.net is newer than cached profile, using it.`);
         }
+      } else {
+        infoLog(
+          'd2-stores',
```
Comment (delphiactual) [inline src/app/inventory/d2-stores.ts:248]: TAG

**Review by delphiactual — APPROVED** [review]

**Pull request merged**